modules/oe_content_event/tests/src/Kernel/EventLinkSourceFilterTest.php

```diff
@@ -110,15 +122,16 @@ public function testEventLinkSourceFilter(): void {
     $future_events = [
       $upcoming_event->id() => $upcoming_event->id(),
       $future_event->id() => $future_event->id(),
+      $ongoing_event->id() => $ongoing_event->id(),
     ];
     $this->assertEqual($query_results, $future_events);
```

Comment:
Actually after introducing the ongoing event the two arrays are not equal: the order is different since the ongoing event should be the first.